R/surv.b.R

```diff
@@ -40,150 +52,237 @@ survClass <- R6::R6Class(
         },
         .run = function() {
             
+            library(tidyr)
```

Comment:
calls to `library()` are verboten!

Reply:
Ok :-) 
I actually didn't use it at the end anyway. I've removed that line.